scripts/calc_rrups_single.sl

```diff
@@ -30,16 +30,21 @@ SRF_FILE=$(getFromYaml ${REL_YAML} srf_file)
 STATION_FILE=$(getFromYaml ${REL_YAML} stat_file)
 FD=$(getFromYaml ${REL_YAML} FD_STATLIST)
 
-OUT_DIR=${REL}/IM_Calc
+OUT_DIR=${REL}/verification
 
 if [[ ! -f ${OUT_DIR}/rrup_${REL_NAME}.csv ]]
 then
+    if [[ ! -d $OUT_DIR ]]; then
+        # create the verification folder if it doesn't exist
+        mkdir $OUT_DIR
```

Comment:
mkdir -p $OUT_DIR will run only make the folder if it doesn't exist. No if needed

Reply:
Thats legit, will change